pehlaschool/Classes/Common/Controls/CompletePopup.cpp

```diff
@@ -14,9 +14,20 @@
 
 namespace CompletePopupNS {
     std::string labelText() {
-        if (LanguageManager::getInstance()->isSwahili())
+        std::string langCode = LanguageManager::getInstance()->customLanguageCode;
+        if (langCode == "en") {
```

Comment:
Shouldn't we use compare() function for string comparison?

Reply:
compare() is for advanced operations which is not required in our case